app.js

```diff
@@ -43,6 +46,10 @@ async function init() {
     queuedUsers = [...allUsers];  
 }
 
-module.exports = {
-    app, init
-};
\ No newline at end of file
+app.listen = function () {
+  init();
+  var server = http.createServer(this)
+  return server.listen.apply(server, arguments)
+}
```

Comment:
Found this pattern in the Express docs as a way of triggering an "initialize" function when the server boots up.